### PR TITLE
dub 1.1.0-beta.1 (devel)

### PR DESCRIPTION
**Pull request by John-Colvin**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Does your submission pass `brew audit --new-formula <formula>` (after doing `brew install <formula>`)?

---

### CODE REVIEW

File: Formula/dub.rb [hunk 1]
```diff
@@ -2,10 +2,10 @@ class Dub < Formula
   desc "Build tool for D projects"
   homepage "https://code.dlang.org/about"
   url "https://github.com/dlang/dub.git",
-      :tag => "v1.0.0",
-      :revision => "b59af2b8befb4fad4157d8c9cc86dba707b2fc87"
+      tag: "v1.0.0",
```
Comment (MikeMcQuaid) [inline Formula/dub.rb]: Please use hash-rockets: https://bot.brew.sh/job/Homebrew%20Core%20Pull%20Requests/8713/version=el_capitan/testReport/junit/brew-test-bot/el_capitan/style_dub/

Apologies that this changed relatively recently.

**Review by MikeMcQuaid — CHANGES_REQUESTED** [review]

File: Formula/dub.rb (hunk 1 above)
Comment (John-Colvin) [inline Formula/dub.rb]: If I do that then `brew audit --new-formula dub` fails, saying "Use the new Ruby 1.9 hash syntax."

Comment (John-Colvin) [inline Formula/dub.rb]: something weird was going on, i was stuck on homebrew `1.0.1`, all fixed now. Have updated the pr to use hash-rockets.

**Review by MikeMcQuaid — APPROVED** [review]

**Comment by MikeMcQuaid** [comment]
Thanks again @John-Colvin!

**Pull request merged**